codespell_lib/data/dictionary.txt

```diff
@@ -3902,6 +3903,7 @@ amerliorators->ameliorators
 amgle->angle
 amgles->angles
 amiguous->ambiguous
+amin->main, admin, amine,
```

Comment:
I think this should stay in `code`; this variable name is very common: amin, bmin, amax, bmax, etc.

Reply:
It's also a numpy thing:
https://numpy.org/doc/stable/reference/generated/numpy.amin.html

Reply:
Indeed. We really need a way to document the typos!

Reply:
Agreed.

Reply:
> Indeed. We really need a way to document the typos!

I dunno if the putting it as the last entry after the comma still works as a reason?

Reply:
Yes, it should work. But then it's probably not as readable as a `# ...` or `// ...` comment.